docs/app/pages/components/components-sections/dashboard/widgets/snippets/app.ts

```diff
@@ -0,0 +1,70 @@
+import {AfterViewInit, Component, TemplateRef, ViewChild} from '@angular/core';
+import {ActionConfig, ActionStatus, DashboardOptions, EnumConfig} from '@ux-aspects/ux-aspects';
+
+@Component({
+    selector: 'app',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.css']
+})
+export class AppComponent implements AfterViewInit {
+    fixedMode: boolean = false;
+    textWidgetEditable: boolean = true;
+    private _dashboardPadding: number = 5;
+
+    get dashboardPadding(): number {
+        return this._dashboardPadding;
+    }
+
+    set dashboardPadding(value: number) {
+        this._dashboardPadding = value;
+
+        this.options = {
+            columns: 4,
+            padding: value,
+            rowHeight: 110,
+            emptyRow: false,
+            minWidth: 187
+        };
+    }
+
+    options: DashboardOptions = {
+        columns: 4,
+        padding: this._dashboardPadding,
+        rowHeight: 110,
+        emptyRow: false,
+        minWidth: 187
+    };
+
+    status: ActionStatus = {label: 'Waiting...', icon: 'radial'};
+
+    actions: ActionConfig[] = [];
+
+    handleActionChange(e: string) {
+        alert(`action clicked: ${e}`);
+    }
```

Comment:
Our convention is for methods to be after class properties and the constructor. We also add return types to functions.